.gitattributes

```diff
@@ -87,13 +87,15 @@
 *.eot               binary
 *.exe               binary
 *.otf               binary
+*.pbm               binary
```

Comment:
Sub module changes need to be reverted. These are undoing latest changes

Reply:
Indeed, this was unintentional. Reverted.